### ISSUE TITLE
Design user experience for search term updates vs. new search

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/dpul-collections#264]
### Summary or User Story

There are two related user stories that we should accommodate in our UI:

1.   As a researcher I need to start a new search after looking through some search results so that I can see results for a different term or phrase.
2.   As a researcher I need to modify my search terms (or a given facet) and retain the same faceting and sorting so that I can continue to hone in on the materials I'm looking for.

We need a UX that accommodates both with clarity and ease of use.

### Acceptance Criteria

- There's a set of wireframes that show how these use cases will be accommodated by the UI
- We have discussed and accepted the wireframes and created an implementation ticket

### First step

Make some drawings

**Comment by sdellis** [comment]
@hackartisan I don't think this use case requires more discussion as you've articulated it. 

However, the details around how different facets are visualized, designed for interaction, and the parameters that are allowed will require more discussion. For example, which facets can have multiple values (i.e., Subjects: "Arts and Culture" AND "Motion Pictures") and which cannot. I don't think we can currently allow multiple Date ranges, for instance. We will need to spend some time on each facet to determine what the behavior will look like and the affordances we can provide for exploration, but those would be in separate tickets. 

I believe the facet values should change as the user is exploring to provide immediate feedback in the form of search results (as opposed to selecting values and then click "Apply"). This exploration path could also leave breadcrumbs/markers that the explorer can use to retrace their steps (aka, "rewind to"), or provide a way to start over from a blank search or one of the "trailheads" that we may provide as starting points (i.e., "Public Domain").

**Comment by hackartisan** [comment]
Initial thoughts for main search results page

![Image](https://github.com/user-attachments/assets/744b8c93-a9a5-4010-8dcd-4d0777c0209b)

**Comment by sdellis** [comment]
@hackartisan this looks good! 

Q: What happens when you clear the sort option from the current filters? Does it sort by Relevance? Or from Year Newest to Oldest? It's not clear to me what the "off" state is.  

And I am not sure I would include sort options in the filters list -- the sort dropdown should be persistent on the search results screen to indicate sort state, so I think having it appear in the list would be redundant and potentially confusing unless you foresee the sort dropdown to be hidden at some point. I think it works for the other facets because there may be several different blocks containing interactive facet UIs that will not be persistent because we can only reasonably display one at a time.

I kind of feel the same about keywords. What if there are multiple keywords? Do they show up in several filter "buttons" or are they all included in one? If the search bar is persistent, then users can directly change the keyword state in the search input field. 

Finally, I would also use the word search "filters" over "parameters".

But these are relatively minor points. I think this mockup will be really helpful and is close to done.

**Comment by hackartisan** [comment]
If sort isn't included in "clear everything" then if I want to do a new search with the default relevance sorting do I have to click "clear everything" then submit a new search then change the sort?

**Comment by tpendragon** [comment]
Next steps are new wireframes given the discussion at https://docs.google.com/document/d/1hHq82XqYs29iSFnJMTvPKoIBMgk6YQM-95w_vTRvGrM/edit?tab=t.0#heading=h.2qctk0gtsjx

**Comment by hackartisan** [comment]
Here are 2 new wireframes

search with "start over" button
<img width="960" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFN5Q0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--af942abeb2714d8551503fd9489b2bf069d7d084/search%20with%20start%20over.jpg" alt="search with start over.jpg" />

search with radio select
<img width="1363" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFd5Q0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1b811e03b7a195c86b16194219610eb441a5d912/search%20with%20checkboxes.jpg" alt="search with checkboxes.jpg" />

**Comment by hackartisan** [comment]
We've decided to prioritize the first use case, and keep the current behavior wherein submitting a search clears the filters that have been applied
